src/Mod/Draft/draftguitools/gui_dimensionstyle.py

```diff
@@ -32,18 +32,78 @@
 from PySide import QtCore
 from . import gui_base
 from draftutils import utils
-from draftobjects.style_dimension import make_dimension_style
+from draftobjects.dimensionstyle import make_dimension_style
 
-class GuiCommandDimensionStyle(gui_base.GuiCommandBase):
+
+'''
+class AnnotationStylesContainer:
+    """The Layer Container"""
+
+    def __init__(self, obj):
+
+        self.Type = "AnnotationContainer"
+        obj.Proxy = self
+
+    def execute(self, obj):
+
+        g = obj.Group
+        g.sort(key=lambda o: o.Label)
+        obj.Group = g
+
+    def __getstate__(self):
+
+        if hasattr(self, "Type"):
+            return self.Type
+
+    def __setstate__(self, state):
+
+        if state:
+            self.Type = state
+
+
+class ViewProviderAnnotationStylesContainer:
+    """A View Provider for the Layer Container"""
+
+    def __init__(self, vobj):
+
+        vobj.Proxy = self
+
+    def getIcon(self):
+
+        return ":/icons/Draft_Annotation_Style.svg"
+
+    def attach(self, vobj):
+
+        self.Object = vobj.Object
+
+    def __getstate__(self):
+
+        return None
+
+    def __setstate__(self, state):
+
+        return None
+
+
+'''
+
+
+
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+class GuiCommandDimensionStyle(gui_base.GuiCommandSimplest):
     """
     The command creates a dimension style object
     """
+    def __init__(self):
+        super().__init__()
+        self.command_name = "DimensionStyle"
```

Comment:
This should be
```py
def __init__(self):
    super().__init__(name="Dimension style")
```
The parent class, GuiCommandSimplest, sets up the command_name attribute.